easybuild/easyconfigs/x/xmitgcm/xmitgcm-0.5.2-foss-2022a.eb

```diff
@@ -0,0 +1,41 @@
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'PythonBundle'
+
+name = 'xmitgcm'
+version = '0.5.2'
+
+homepage = 'https://github.com/MITgcm/xmitgcm'
+description = """xmitgcm is a python package for reading MITgcm binary 
+MDS files into xarray data structures. By storing data in dask arrays, 
+xmitgcm enables parallel, out-of-core analysis of MITgcm output data."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('MITgcmutils', '0.1.2'),
+    ('dask', '2022.10.2'),
```

Comment:
There is already `scikit-image-0.19.3-foss-2022a.eb` depending on dask v2022.10.0 in this toolchain. Please downgrade this dependency to that version.

Reply:
Done. I downgraded `dask`to the already existing `2022.10.0` version.